app/views/spree/user_mailer/confirmation_instructions.html.haml

```diff
@@ -13,3 +13,6 @@
 = render 'shared/mailers/signoff'
 
 = render 'shared/mailers/social_and_contact'
+
+%p.notice
+  = t :email_confirmation_notice_unexpected, sitename: @instance, contact: @contact
```

Comment:
`ContentConfig.footer_email` is an optional field that may not be set. It's annoying, but I think we need two messages here. One paragraph with contact email and one without.

Reply:
Ah, I wasn't aware of that default. Thank you.